server/src/main/scala/controllers/application/WithExports.scala

```diff
@@ -123,234 +95,52 @@ trait WithExports {
             Result(
               ResponseHeader(200, Map("Content-Disposition" -> s"attachment; filename=$fileName.csv")),
               HttpEntity.Strict(ByteString(csvData), Option("application/csv")
+                                )
               )
-            )
 
           case None =>
             log.error(s"Missing headline data for ${startOfWeekMidnight.ddMMyyString} for Terminal $terminal")
             NotFound(s"Sorry, no headlines available for week starting ${startOfWeekMidnight.ddMMyyString}")
         }
       }
     }
-
   }
 
-  def exportApi(day: Int, month: Int, year: Int, terminalName: String): Action[AnyContent] = authByRole(ApiViewPortCsv) {
-    val terminal = Terminal(terminalName)
-    Action.async { _ =>
-      val startHour = 0
-      val endHour = 24
-      val dateOption = Try(SDate(year, month, day, 0, 0)).toOption
-      val terminalNameOption = airportConfig.terminals.find(_ == terminal)
-      val resultOption = for {
-        date <- dateOption
-        terminalName <- terminalNameOption
-      } yield {
-        val pit = date.millisSinceEpoch
-        val portStateForPointInTime = loadBestPortStateForPointInTime(pit, terminalName)
-        val fileName = f"export-splits-${portCode.toString}-$terminalName-${date.getFullYear()}-${date.getMonth()}-${date.getDate()}"
-        flightsForCSVExportWithinRange(terminalName, date, startHour = startHour, endHour = endHour, portStateForPointInTime).map {
-          case Some(csvFlights) =>
-            val csvData = CSVData.flightsWithSplitsWithAPIActualsToCSVWithHeadings(csvFlights)
-            Result(
-              ResponseHeader(OK, Map(
-                CONTENT_LENGTH -> csvData.length.toString,
-                CONTENT_TYPE -> "text/csv",
-                CONTENT_DISPOSITION -> s"attachment; filename=$fileName.csv",
-                CACHE_CONTROL -> "no-cache")
-              ),
-              HttpEntity.Strict(ByteString(csvData), Option("application/csv"))
-            )
-          case None => NotFound("No data for this date")
-        }
-      }
-      resultOption.getOrElse(
-        Future(BadRequest("Invalid terminal name or date"))
-      )
+  class SummaryActor extends Actor {
```

Comment:
possibly worth moving this into its own file, with the other actors.

Reply:
sorry that's actually dead code from when i started the implementation. i'll remove it